packages/commonwealth/server/migrations/20231027172817-chain-semantics-update.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.renameTable('Chains', 'Communities', {
+        transaction,
+      });
+      await queryInterface.sequelize.query(
+        `
+        ALTER INDEX "Chains_pkey" RENAME TO "Communities_pkey";
+      `,
+        { transaction }
+      );
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.sequelize.query(
+        `
+        DROP VIEW "Chains";
+      `,
+        { transaction }
+      );
```

Comment:
Tested functionality and works fine, but the down migration fails since the `Chains` view doesn't exist. This should be removed?

Reply:
Nice catch. Removed.